models/Product.js

```diff
@@ -0,0 +1,5 @@
+export default class Product {
+  constructor () {
+    console.log('Product module');
```

Comment:
Remove `console.log` and define `sayHello()` that write message into console by `console.log()`.

Reply:
Done.